CanDIG/katsu
# delete a dataset api

What's new:
- remove the old delete API
- add new delete dataset api
- fix the bug that katsu container cannot restart
- bump katsu version to 2.1.0 since the ingest API change

How to test:
- confirm delete API works by calling from ingest repo
- Need to update https://github.com/CanDIG/candigv2-ingest/pull/20
